### ISSUE TITLE
What's the reason it cannot simply be installed as a plugin on Linux?

### ISSUE THREAD

**Issue by Lord-Vlad** [Acly/krita-ai-tools#12]
I like the convenience of having Krita installed through the package manager of my choice, that's why I am curious about what's the reason behind the patching. For Windows, it seems to work just fine like any other plugin.

**Comment by Acly** [comment]
No reason why it shouldn't work. You just need to put the files in the right places.

For Windows it's easy because all installations are isolated and have the same structure, which is replicated in the .zip.

For Linux you have some people using AppImage, some Flatpak, and then various distros are free to distribute the files in different places. Krita from package managers is often a bit outdated, and the plugin only works for the latest version, hence the instructions focus on the AppImage.

**Comment by Lord-Vlad** [comment]
Thanks for the answer.

**Comment by ShadwDrgn** [comment]
if I were to do it via package manager how would the directory structure work? I've installed other plugins by simply copying the entire zip to a folder in pykrita. Is that what I'd do here? this seems to have a lib folder which i'm not familiar with. I'm not seeing a lib folder anywhere in .local/share/krita which seems to be where my krita files are and is where I installed krita-ai-diffusion as an example.
